<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>time</string>
   </property>
   <property name="text">
    <string>00:00/00:00</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="toolButton_last">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;last&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/gtk-media-next-rtl.png</normaloff>images/gtk-media-next-rtl.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_playpause">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;play&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/gtk-media-play-ltr.png</normaloff>images/gtk-media-play-ltr.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_stop">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;pause&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/gtk-media-stop.png</normaloff>:/images/gtk-media-stop.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_next">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;next&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/gtk-media-next-ltr.png</normaloff>images/gtk-media-next-ltr.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>125</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QToolButton" name="toolButton_open">
      <property name="toolTip">
       <string>open</string>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_list">
      <property name="toolTip">
       <string>list</string>
      </property>
      <property name="text">
       <string>list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_lrc">
      <property name="toolTip">
       <string>lrc</string>
      </property>
      <property name="text">
       <string>lrc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>350</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>歌词</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>音乐</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>音乐路径</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>歌词路径</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox_playMode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>顺序播放</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>全部循环</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单曲循环</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>随机播放</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
